holidata/holidays/hr-HR.py

```diff
@@ -7,18 +7,19 @@
 class hr_HR(Locale):
     """
     01-01: [NF] Nova Godina
-    01-06: [NRF] Bogojavljanje / Sveta tri kralja
+    01-06: [NRF] Bogojavljanje ili Sveta tri kralja
     05-01: [NF] Praznik rada
     06-22: [NF] Dan antifašističke borbe
     06-25: [NF] Dan državnosti
-    08-05: [NF] Dan domovinske zahvalnosti
+    08-05: [NF] Dan pobjede i domovinske zahvalnosti i Dan hrvatskih branitelja
```

Comment:
OK: This holiday has this name since 2008, before it was just "Dan pobjede i domovinske zahvalnosti" (accorging to https://hr.wikipedia.org/wiki/Dan_pobjede_i_domovinske_zahvalnosti,_Dan_hrvatskih_branitelja).

So we would strictly need a function to account for the name change. However since holidata currently only serves data from 2011, we can leave it like this.